pom.xml

```diff
@@ -115,7 +115,7 @@
         <version.slf4j>2.0.12</version.slf4j><!-- 2.0.12 is approved #13344 -->
         <version.sqlite-jdbc>3.42.0.0</version.sqlite-jdbc><!-- 3.42.0.0 is approved in #9089 -->
         <!-- note, when upgrading, the field LibSumoAmbassador#VALID_LIBSUMO_VERSIONS needs to be changed too -->
-        <version.libsumo>1.18.0</version.libsumo>
```

Comment:
SUMO 1.20.0 libsumojni.jar is not yet available, therefore currently only upgrade LibSumoAmbassador to 1.19.0, but tested already with 1.20.0-SNAPSHOT.

Reply:
Should we wait until 1.20.0 is available before merging this PR?

Reply:
I asked again!

Reply:
They added 1.20.0 to the repository already: https://repo.eclipse.org/content/repositories/sumo-releases/org/eclipse/sumo/libsumo/